### PR TITLE
Sassdoc pages will no longer appear in the menu twice

### LINKED ISSUE #7: Sassdoc-pages appear in the menu twice
image

### PR DESCRIPTION
**Pull request by elessar-ch**
Fixes #7

### CODE REVIEW

File: lib/metalsmith-sitemap.js [hunk 1]
```diff
@@ -41,24 +42,24 @@ const metalsmithSitemap = function() {
...
+    // for (const name in metadata.sassdoc.comments) {
+    //   const comment = metadata.sassdoc.comments[name] 
+    //   sassdocNode.children.push({
+    //     name: name,
+    //     render: true,
+    //     filename: 'sassdoc/' + comment.urlFriendlyName,
+    //     title: name,
+    //     order: 0,
+    //   })
+    // }
```
Comment (davidknezic) [inline lib/metalsmith-sitemap.js]: how comes this is commented? should we remove this?

File: docs/wooooo.hbs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: wooooo
+draft: true
 ---
 {{#> page}}
   {{#*inline "title"}}{{title}}{{/inline}}
```
Comment (davidknezic) [inline docs/wooooo.hbs:6]: what is what `woooo` page anyways? do we need that?

File: docs/components/wooooooooo.hbs [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: wooooo
+draft: true
 ---
 {{#> page}}
   {{#*inline "title"}}{{title}}{{/inline}}
```
Comment (davidknezic) [inline docs/components/wooooooooo.hbs]: could we remove this page?

**Review by davidknezic — CHANGES_REQUESTED** [review]

File: docs/wooooo.hbs (hunk 2 above)
Comment (elessar-ch) [reply docs/wooooo.hbs:6]: That's currently the only page showing how to include a snippet and a demo in a handlebars page

File: docs/components/wooooooooo.hbs (hunk 3 above)
Comment (elessar-ch) [reply docs/components/wooooooooo.hbs]: Will do

File: lib/metalsmith-sitemap.js (hunk 1 above)
Comment (elessar-ch) [reply lib/metalsmith-sitemap.js]: Will do

**Comment by elessar-ch** [comment]
Have a look again @davidknezic

**Review by davidknezic — APPROVED** [review]

**Pull request merged**